moped-editor/src/views/projects/projectView/ProjectMilestones/MilestoneAutocompleteComponent.js

```diff
@@ -30,6 +42,12 @@ const MilestoneAutocompleteComponent = ({
       field,
       value: newValue ?? null,
     });
+    // Also update the moped_milestone field aka the Related Phase
+    apiRef.current.setEditCellValue({
+      id,
+      field: "moped_milestone",
+      value: { related_phase_id: relatedPhaseLookup[newValue] },
+    });
```

Comment:
@chiaberry I tested this again, and I think that this field and value are making their way into the payload when saving a new row.

Reply:
sorry to pile on, but I am seeing this graphql error when i try to add a milestone:
<img width="384" alt="Screenshot 2024-11-05 at 10 23 07 AM" src="https://github.com/user-attachments/assets/b066bfd0-5930-40d0-bf2d-588c4648cce0">

Reply:
yep, thats the error Mike is pointing out -- thanks I will address this morning!

Reply:
I think i fixed it -- i was so focused on existing rows, i didnt go back to check on adding new rows

Reply:
thanks for fixing! 🙌